docs/guide/cli.md

```diff
@@ -90,7 +90,17 @@ Certain files might need to be changed during the preparation of a build
 job. In that situation, `lage` can ignore those files when calculating what
 has changed with the `--since` flag.
 
-  
+#### include-dependencies
+_type: boolean_
+
+Include all transitive dependencies when running a command(s).
+This is useful for situations where you want to "set up" a package that relies on other packages being set up.
+
+```
+lage setup --scope my-package --include-dependencies
+# my-package and all of its dependencies will be setup
+```
+
```

Comment:
How is this different from using `--to`?

Reply:
`--to` scopes running tasks to specific package(s) but does not necessarily run them for their dependencies.

E.g, given the config:

```js
  pipeline: {
    lint: ["build"],
    prepare: ["^prepare"],
    build: ["prepare", "^build"],
    test: ["build"]
  }
```

Using `--to` for the `prepare` and `build` tasks would run the tasks for the dependencies since their declare tasks dependencies include running that task on their dependencies.

However, `lint` and `test` would only run the task for the scoped packages **without** running the task for the dependencies.